hopsworks-common/src/main/java/io/hops/hopsworks/common/provenance/util/ProvHelper.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * This file is part of Hopsworks
+ * Copyright (C) 2019, Logical Clocks AB. All rights reserved
+ *
+ * Hopsworks is free software: you can redistribute it and/or modify it under the terms of
+ * the GNU Affero General Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later version.
+ *
+ * Hopsworks is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ */
+package io.hops.hopsworks.common.provenance.util;
+
+import io.hops.hopsworks.common.provenance.core.ProvParser;
+import io.hops.hopsworks.common.provenance.util.functional.CheckedFunction;
+import io.hops.hopsworks.exceptions.ElasticException;
+import io.hops.hopsworks.exceptions.ProvenanceException;
+import io.hops.hopsworks.restutils.RESTCodes;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+
+public class ProvHelper {
```

Comment:
Any reason why this class has all static methods and it's not a java ee bean?

Reply:
This is just a Helper class used to parse elastic results. I used it in two other DTOs. It is not supposed to be a bean.